planning/behavior_path_planner/include/behavior_path_planner/scene_module/avoidance/avoidance_module.hpp

```diff
@@ -200,6 +206,7 @@ class AvoidanceModule : public SceneModuleInterface
   void fillObjectMovingTime(ObjectData & object_data) const;
   void compensateDetectionLost(
     ObjectDataArray & target_objects, ObjectDataArray & other_objects) const;
+  void fillAvoidancePath(AvoidancePlanningData & data, DebugData & debug) const;
```

Comment:
Is this function generating the avoidance path? I don't get a point of what "fill" means.

Reply:
You're right :+1: 
I think `fillShiftLine` is better than `fillAvoidancePath`. What do you think ?

Reply:
fix in https://github.com/autowarefoundation/autoware.universe/pull/2533/commits/599a08cb891daa8283b045b68a47a9eb5d845618

Reply:
Nice. Thank you